source/jobs/creating-jobs.txt

```diff
@@ -96,16 +96,43 @@ Steps
 
   .. step:: Enter the source MongoDB connection details 
 
-      On the :guilabel:`Connect Destination DB` form, input your 
-      :guilabel:`MongoDB connection string (URI)`.
+      You can provide the MongoDB connection details by selecting an Atlas
```

Comment:
Consider rewrite in form "To provide XXX, select ...."

Reply:
Updated.